src/prefabs/pageWithEditDataTable.js

```diff
@@ -0,0 +1,11725 @@
+                              value: 0,
+                            },
+                          },
+                        },
+                        {
+                          value: false,
+                          label: 'Styles',
+                          key: 'styles',
+                          type: 'TOGGLE',
+                        },
+                        {
+                          type: 'COLOR',
+                          label: 'Background color',
+                          key: 'backgroundColor',
+                          value: 'White',
+                          configuration: {
+                            condition: {
+                              type: 'SHOW',
+                              option: 'styles',
+                              comparator: 'EQ',
+                              value: true,
+                            },
+                          },
+                        },
+                        {
+                          type: 'COLOR',
+                          label: 'Border color',
+                          key: 'borderColor',
+                          value: 'Accent1',
+                          configuration: {
+                            condition: {
+                              type: 'SHOW',
+                              option: 'styles',
+                              comparator: 'EQ',
+                              value: true,
+                            },
+                          },
+                        },
+                        {
+                          type: 'COLOR',
+                          label: 'Border color (hover)',
+                          key: 'borderHoverColor',
+                          value: 'Black',
+                          configuration: {
+                            condition: {
+                              type: 'SHOW',
+                              option: 'styles',
+                              comparator: 'EQ',
+                              value: true,
+                            },
+                          },
+                        },
+                        {
+                          type: 'COLOR',
+                          label: 'Border color (focus)',
+                          key: 'borderFocusColor',
+                          value: 'Primary',
+                          configuration: {
+                            condition: {
+                              type: 'SHOW',
+                              option: 'styles',
+                              comparator: 'EQ',
+                              value: true,
+                            },
+                          },
+                        },
+                        {
+                          value: false,
+                          label: 'Hide label',
+                          key: 'hideLabel',
+                          type: 'TOGGLE',
+                          configuration: {
+                            condition: {
+            
```

Comment:
Your new `autoOK` option needs to be added here as well

Reply:
Added `autoOk` option